scripts/epoch.bat

```diff
@@ -0,0 +1,5 @@
+SETLOCAL ENABLEEXTENSIONS
+
+SET REL_NAME=aeternity
+
+call %~dp0/%REL_NAME% %*
```

Comment:
One discussion at https://github.com/aeternity/aeternity/pull/2062 is settled, can you please add deprecation warning matching that one. Cc @dincho

Reply:
I added the warning.